docs/sources/features/datasources/mssql.md

```diff
@@ -70,11 +71,6 @@ Example:
 
 Make sure the user does not get any unwanted privileges from the public role.
 
-### Known Issues
```

Comment:
I think it would be nice to leave this section, but use the same description as in tooltip and some general recommendation regarding service packs, for example:

> If you're using an older version of Microsoft SQL Server like 2008 and 2008R2 you may need to disable encryption to be able to connect. If possible, we recommend you to use the latest service pack available for optimal compatibility.